### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by Id3aFly**
When using the API first it was not clear at all how to use it. Here is a proposal that tries to improve the documentation.

I've seen both of the modified strings to also be part of index.adoc and generated/html/index.html. These should be potentially updated accordingly (or let me know how to do it)

### CODE REVIEW

File: backend/lib/app/validate.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const presets = {
 
 const swaggerInfo = {
   description:
-    'This endpoint is intended to validate a document against the specified tests.',
+    'This endpoint is intended to validate a document against the specified tests. In the list of tests you need to provide at least one dict, where each dict is used to run either a single test or an entire preset. For \'name\' provide the test\'s or the preset\'s name, and as \'type\' provide either \'test\' or \'preset\'. For the value of the document-property just provide the json of your CSAF document.',
```
Comment (tschmidtb51) [inline backend/lib/app/validate.js:33]:
```suggestion
    'This endpoint is intended to validate a document against the specified tests. In the list of tests provide at least one object, where each object is used to run either a single test or an entire preset. For \'name\' provide the test\'s or the preset\'s name, and as \'type\' provide accordingly either \'test\' or \'preset\'. For the value of the property \'document\' just provide the json of your CSAF document.',

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please also regenerate the docs using one of the methods described here: https://openapi-generator.tech/docs/installation/

We will automate that later in the process (see also #34).

**Comment by Id3aFly** [comment]
Regarding the openapi-generator: It requires a petstore.yaml. I did not find one in the repository. Can you share it maybe so I don't have to create it anew?

**Comment by tschmidtb51** [comment]
@domachine Please have a look whether we forgot that file or there is a different way to generate the documentation... Please comment here.

**Comment by domachine** [comment]
Sorry, I can not find a reference to a file called `petstore.yaml` in the project. This sounds more like a sample file.

**Comment by Id3aFly** [comment]
And do you use any other yaml file for this? If not I can try later to run the openapi-generator without this file.

**Comment by tschmidtb51** [comment]
Changed base to be able to edit things...

**Pull request merged**